docker-compose.yml

```diff
@@ -154,7 +157,45 @@ services:
       - 24224:24224
       - 24224:24224/udp
     depends_on:
-      - elasticsearch
+      - kibana
     volumes:
       - ./fluent-bit.conf:/fluent-bit/etc/fluent-bit.conf
 
+  cserver:
+    image: quay.io/packet/cacher:workflow
```

Comment:
The setup should not be using Cacher.

Reply:
Yes, I know that and I will remove this once we get the confirmation about cacher from packet. on whether we should replace cacher with tinkerbell/hardware package.